rllib/algorithms/algorithm_config.py

```diff
@@ -1784,9 +1768,6 @@ def training(
                 dashboard. If you're seeing that the object store is filling up,
                 turn down the number of remote requests in flight, or enable compression
                 in your experiment of timesteps.
-            _enable_learner_api: Whether to enable the LearnerGroup and Learner
```

Comment:
~nit: don't remove the doc for it. Just add that it has been replaced with `_enable_new_stack_api`~

I see that you throw the deprecation warning right away.